drivers/gpio/gpio_stm32.c

```diff
@@ -456,6 +456,31 @@ static int gpio_stm32_init(struct device *device)
 	return 0;
 }
 
+static int gpio_stm32_afio_init(struct device *device)
+{
+	UNUSED(device);
+
+
+#if defined(CONFIG_SOC_SERIES_STM32F1X)
+
+	LL_APB2_GRP1_EnableClock(LL_APB2_GRP1_PERIPH_AFIO);
+
+	#if defined(CONFIG_GPIO_STM32_SWJ_ENABLE)
+	__HAL_AFIO_REMAP_SWJ_ENABLE();
+	#elif defined(CONFIG_GPIO_STM32_SWJ_NONJTRST)
+	__HAL_AFIO_REMAP_SWJ_NONJTRST();
```

Comment:
Can you add comments stating the released pins?

Reply:
![2019-07-31](https://user-images.githubusercontent.com/12996088/62202388-c9f45500-b3bb-11e9-911e-8168926276fe.jpg)
 in page 1081 of the manual.

Reply:
So putting this information in the code (for each case, which pins are released) would be nice